### PR TITLE
Adding readings

### PR DESCRIPTION
**Pull request by davidabrahams**
I added readings assignments to all the HWs

### CODE REVIEW

File: day01/README.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ The tests you will be running on your `MyArrayList` class will be using this `Co
 
 # Assignment
 
+## Reading
+
+- In *Think Algorithms*, read sections **3.3, 4.3, and 4.4**
```
Comment (thecardkid) [inline day01/README.md]: Don't think 3.3 is necessary here. Instructions in his book are mostly wrong now. We can write our own blurb.

Also not sure about 4.4, as it pretty much gives away all implementation details? But then again we're checking in with them afterwards so maybe that's okay. Thoughts?

File: day05/README.md [hunk 2]
```diff
@@ -16,6 +12,12 @@ After completing this assignment you should be able to:
 
 ## Assignment
 
+### Reading
+
+- In *Think Algorithms*, read sections **14.5, 14.6, 15.4 and 15.5.**.
```
Comment (thecardkid) [inline day05/README.md]: Can you include 14.4, which presents the need for hashing.

Comment (davidabrahams) [inline day05/README.md]: Sure. This seems a bit long though. Ideas for sections we could cut?

File: day01/README.md (hunk 1 above)
Comment (davidabrahams) [inline day01/README.md]: Oh this was supposed to be 3.4. Oops good catch.

**Comment by davidabrahams** [comment]
@thecardkid all your changes have been addressed

File: day01/README.md (hunk 1 above)
Comment (kghite) [inline day01/README.md]: Agreed on 3.3, Sidd's instructions seem fine.  Ch. 4 is pretty good though.  Could just say read after the assignment is complete or if you get confused?  They should definitely read the performance analysis either way.

**Pull request merged**